opam

```diff
@@ -5,9 +5,12 @@ homepage: "https://github.com/ocamllabs/higher"
 bug-reports: "https://github.com/ocamllabs/higher/issues"
 dev-repo: "https://github.com/ocamllabs/higher.git"
 license: "MIT"
-build: [
-  "ocaml" "pkg/build.ml" "native=%{ocaml-native}%"
-                         "native-dynlink=%{ocaml-native-dynlink}%"
+build:
+[[ "ocaml" "pkg/pkg.ml" "build"
+           "--installer" "true" ]]
```

Comment:
Since this has not been merged yet, I'll simply comment here. With the latest topkg you have to replace this line 
with `"--pinned" "%{pinned}%"`. See https://github.com/dbuenzli/topkg/issues/44 if you want to know why.